app.js

```diff
@@ -12,6 +12,10 @@ app.get(
   "/api/articles/:article_id/comments",
   controller.articles.getArticleCommentsById
 );
+app.post(
+  "/api/articles/:article_id/comments",
+  controller.articles.postArticleCommentById
```

Comment:
please require this function in at the top of your file, to save you having to retype controller.articles every time!